scaleway/resource_rdb_instance.go

```diff
@@ -611,16 +611,11 @@ func resourceScalewayRdbInstanceUpdate(ctx context.Context, d *schema.ResourceDa
 			return diag.FromErr(err)
 		}
 
-		// get endpoints to detach. It will handle only private networks
-		endPointsToRemove, err := endpointsToRemove(res.Endpoints, d.Get("private_network"))
```

Comment:
`endpointsToRemove` get the list of endpoints that should be removed, this makes so we don't remove endpoints that should not.
Later on, you changed the loop to delete all endpoints that container a private_network. I don't understand the goal here

Reply:
Maybe the `endpointsToRemove` should just be fixed with new changes to detect.
Also, if you remove it, it becomes unused. Why should we stop using it with the many tests that were made.
https://github.com/scaleway/terraform-provider-scaleway/blob/f52c7bc509522e653724c48df6e53aa4ee202c4f/scaleway/helpers_rdb_test.go#L14-L139

Reply:
This method turns useless in order to check only one private network at once. It's removed on the last commit